ui/app/AppLayouts/Communities/popups/TransferOwnershipPopup.qml

```diff
@@ -110,8 +110,8 @@ StatusDialog {
                 onClicked: {
                     // Pre-populated dialog with the relevant Owner token info:
                     const store = WalletStores.RootStore.currentActivityFiltersStore
-                    const uid = store.collectiblesList.getUidForData(token.key, token.tokenAddress, token.chainId);
-                    root.transferOwnershipRequested(uid, token.accountAddress)
+                    const uid = store.collectiblesList.getUidForData("0", token.tokenAddress.toLowerCase(), token.chainId);
```

Comment:
what does "0" mean here?
Maybe it's worth standardizing the owner token ID if this is what it represents. A constant property would be enough for now.

Reply:
this representing the tokenId for the owner token, but I am not sure why it is 0. Do you know who would have an idea about this? The code is so confusing and I am lost.

Reply:
Hmm, after looking at the code, I'm not sure either anymore :grimacing:

Reply:
(I mean, what the `0` stands for)

Reply:
not spending anymore time on it. it works. we can come back to this at a future time